test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -157,20 +157,51 @@ func TestNamespaceScheduler(t *testing.T) {
 					GenerateName: "e2e-nss-2-",
 				},
 				Spec: clusterv1alpha1.ClusterSpec{
-					KubeConfig: string(server2Kubeconfig),
+					KubeConfig:    string(server2Kubeconfig),
+					Unschedulable: true, // cordoned.
 				},
 			}, metav1.CreateOptions{})
 			require.NoError(t, err, "failed to create cluster2")
 
-			t.Log("Delete the old cluster and expect the namespace to end up scheduled to the new cluster.")
+			t.Log("Delete the old cluster and expect the namespace to end up unschedulable (since the new cluster is cordoned).")
 
 			err = clusterClient.Delete(ctx, cluster1.Name, metav1.DeleteOptions{})
 			require.NoError(t, err, "failed to delete cluster1 %q", cluster1.Name)
 
+			err = expect(namespace, unschedulableMatcher())
+			require.NoError(t, err, "did not see namespace marked unschedulable")
+
+			t.Log("Uncordon the new cluster and expect the namespace to end up scheduled to it.")
+
+			cluster2.Spec.Unschedulable = false // uncordoned.
```

Comment:
(No action required) Would `Cordoned` be a better name for the field than `Unschedulable`?

Reply:
Maybe. I wanted to follow the precedent for Nodes, but we've already broken with that tradition by adding `EvictAfter` (or API-level eviction at all; node eviction/"drain" seems to be entirely client-side)

And in any case, even Nodes aren't consistent -- the API is `unschedulable bool`, but the CLI is `kubectl cordon <node>`